<launch>
  <node pkg="rr_platform" type="bigoli_motor_relay_node" name="bigoli_motor_relay_node" output="screen" respawn="false">
    <param name="speed_topic" type="string" value="/speed" />
    <param name="steering_topic" type="string" value="/steering" />
    <param name="serial_port" type="string" value="/dev/ttyACM0" />
    <param name="kP" type="double" value= "0.05" />
    <param name="kI" type="double" value= "0.00" />
    <param name="kD" type="double" value= "0.08" />
    <param name="trim" value= "4" />
  </node>
</launch>
